### PR TITLE
Include process.env.PYTHONPATH when creating PYTHONPATH in pytestTestRunner.ts

### PR DESCRIPTION
**Pull request by bergkvist**
Fixes: https://github.com/kondratyev-nv/vscode-python-test-adapter/issues/230

### CODE REVIEW

**Comment by bergkvist** [comment]
The failing tests seem to have failed with some kind of timeout, which seems unrelated to the change I made:
```
Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (d:\a\1\s\out\test\tests\pytestScript.test.js)
Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (d:\a\1\s\out\test\tests\pytestScript.test.js)
	at listOnTimeout (internal/timers.js:549:17)
	at processTimers (internal/timers.js:492:7)
```

**Comment by bergkvist** [comment]
The failing tests seem to have failed with some kind of timeout:
```
Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (d:\a\1\s\out\test\tests\pytestScript.test.js)
Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (d:\a\1\s\out\test\tests\pytestScript.test.js)
	at listOnTimeout (internal/timers.js:549:17)
	at processTimers (internal/timers.js:492:7)
```

File: src/pytest/pytestTestRunner.ts [hunk 1]
```diff
@@ -157,6 +157,7 @@ export class PytestTestRunner implements ITestRunner {
         const updatedPythonPath = [
             config.getCwd(),
             environment.PYTHONPATH,
```
Comment (kondratyev-nv) [inline src/pytest/pytestTestRunner.ts:159]: nit: Can you please rename `environment` here to something like `envFileEnvironment` so there won't be any confusion with the whole env?

**Review by kondratyev-nv — APPROVED** [review]
Thanks! Don't worry about the timeout. I see this is only happening in Azure on Windows. So it's definitely not because of your changes.

Do you think you can add some tests for that fix?

**Comment by bergkvist** [comment]
I don't seem to be able to run the test suite locally:
```
$ npm test                                                                                                                                (bergkvist/fix-pythonpath) 

> vscode-python-test-adapter@0.6.7 pretest /home/tobias/repos/vscode-python-test-adapter
> npm run build


> vscode-python-test-adapter@0.6.7 build /home/tobias/repos/vscode-python-test-adapter
> tsc


> vscode-python-test-adapter@0.6.7 postbuild /home/tobias/repos/vscode-python-test-adapter
> copyfiles "resources/python/**/*.py" out


> vscode-python-test-adapter@0.6.7 test /home/tobias/repos/vscode-python-test-adapter
> cross-env SOME_PROCESS_VARIABLE=HelloFromProcessEnv cross-env CODE_TESTS_WORKSPACE=./test/test_samples/samples-workspace.code-workspace node ./out/test/vscode-runner.js

Python executable is python
Using python /usr/bin/python 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0]

Found .vscode-test/vscode-1.55.2. Skipping download.
The futex facility returned an unexpected error code.

Exit code:   null
Done

SIGABRT
Failed to run tests
npm ERR! Test failed.  See above for more details.
```

**Comment by kondratyev-nv** [comment]
> The futex facility returned an unexpected error code.

Weird... Never saw this issue before. 
Well, I'll merge it and try to add some tests by myself. Thanks for working on this!

**Comment by bergkvist** [comment]
Okay, thanks @kondratyev-nv :)

**Pull request merged**